[23-Aug-2022 18:00:20 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:55:40 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:55:40 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:55:40 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:55:48 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:56:04 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:56:06 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:56:13 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:56:20 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[24-Aug-2022 10:56:20 Asia/Tehran] PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 5.6.40. in /home3/clientde/marco.clientdemo.ir/vendor/composer/platform_check.php on line 25
[27-Aug-2022 18:57:09 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 18:58:04 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 18:58:39 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:04:31 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:25 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:26 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:28 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:29 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:29 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
[27-Aug-2022 19:15:31 Asia/Tehran] PHP Fatal error:  Uncaught Error: Call to undefined function Symfony\Component\ErrorHandler\ErrorRenderer\highlight_file() in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php:254
Stack trace:
#0 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/trace.html.php(36): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->fileExcerpt('/home3/clientde...', 254, 5)
#1 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#2 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/traces.html.php(44): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/trace.htm...', Array)
#3 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#4 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception.html.php(58): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/traces.ht...', Array)
#5 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#6 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/Resources/views/exception_full.html.php(35): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#7 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(336): include('/home3/clientde...')
#8 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(146): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->include('views/exception...', Array)
#9 /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php(72): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->renderException(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#10 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(599): Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer->render(Object(Symfony\Component\ErrorHandler\Exception\FlattenException))
#11 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(573): Illuminate\Foundation\Exceptions\Handler->renderExceptionWithSymfony(Object(Error), true)
#12 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\Foundation\Exceptions\Handler->renderExceptionContent(Object(Error))
#13 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(533): Illuminate\Foundation\Exceptions\Handler->convertExceptionToResponse(Object(Error))
#14 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(447): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Error))
#15 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(366): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Error))
#16 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(220): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Error))
#17 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(197): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Error))
#18 /home3/clientde/marco.clientdemo.ir/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(257): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#19 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#20 {main}
  thrown in /home3/clientde/marco.clientdemo.ir/vendor/symfony/error-handler/ErrorRenderer/HtmlErrorRenderer.php on line 254
